src/main/java/seedu/address/model/util/WeightUnit.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.model.util;
+
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+
+/**
+ * Utility class to represent unit of weight.
+ */
+public class WeightUnit {
+    public static final String KILOGRAM = "kg";
+    public static final String POUND = "lb";
+    public static final String MESSAGE_INVALID_UNIT_STATUS = "Weight must be in kg or lb.";
+    public static final double KILOGRAM_TO_POUND_MULTIPLIER = 2.20462;
+    private final String weightUnit;
+
+    /**
+     * Constructs a Default WeightUnit of kg.
+     */
+    public WeightUnit() {
+        this.weightUnit = KILOGRAM;
+    }
+
+    /**
+     * Constructs a {@code WeightUnit}.
+     *
+     * @param weightUnit A valid weightUnit of either kg or lb.
+     */
+    public WeightUnit(String weightUnit) {
+        checkArgument(isValidUnit(weightUnit), MESSAGE_INVALID_UNIT_STATUS);
+        this.weightUnit = weightUnit;
+    }
+
+    /**
+     * Returns true if a given String value is a valid unit.
+     */
+    public static boolean isValidUnit(String unit) {
+        return unit.equals(KILOGRAM) | unit.equals(POUND);
+    }
+
+    /**
+     * Returns true if a given String value represents pound unit.
+     */
+    public boolean isPoundUnit() {
+        return this.weightUnit.equals(POUND);
+    }
+
+    public static double getKgInPound(double weightInKilogram) {
+        checkArgument(weightInKilogram > 0);
```

Comment:
shd this call isValidWeight?

Reply:
I made WeightUnit check this on its own to avoid cyclic dependency